transports/webrtc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-webrtc"
-version = "0.1.0-alpha"
+version = "0.4.0-alpha"
```

Comment:
Note that this is necessary, given that there is already `v0.3.0` under the `libp2p-webrtc` crate on crates.io. See https://github.com/libp2p/rust-libp2p/pull/3163.

//CC @melekes